packages/core/core/src/Transformation.js

```diff
@@ -144,8 +147,9 @@ export default class Transformation {
         logger.verbose([
           {
             origin: '@parcel/core',
-            message: `Could not load existing source map for ${escapeMarkdown(
-              path.relative(this.options.projectRoot, asset.value.filePath),
+            message: md`Could not load existing source map for ${path.relative(
+              this.options.projectRoot,
```

Comment:
At some point we should make sure these are relative to the CWD not the project root...